backend/StreamServerApp/models.py

```diff
@@ -1,6 +1,20 @@
 from django.db import models
 
 
+class Series(models.Model):
+    title = models.CharField(max_length=200)
```

Comment:
We'll have to find a way to attribute a thumbnail to the Series as well. Maybe pick a random one ?

Reply:
Ok. Would you like this random pick to be done once when populating the db, or every time we would need a thumbnail?

Reply:
I guess it's ok if we do it once. But maybe we need one per season (if we are using a carrousel for seasons as well).

Reply:
Ok.
I think in the end we will not use a carousel per season. See the original issue for the discussion.